share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -461,7 +463,7 @@ e4s-oneapi-build:
     SPACK_CI_STACK_NAME: e4s-power
 
 e4s-power-generate:
-  extends: [ ".e4s-power", ".generate-x86_64", ".e4s-power-generate-tags-and-image"]
+  extends: [ ".e4s-power", ".generate", ".e4s-power-generate-tags-and-image"]
```

Comment:
I feel like this probably still needs to extend the arch-specific generate definition